docs/reference/Scheduler.md

```diff
@@ -301,12 +301,12 @@ It might be important to understand the basics of kubernetes before deep diving
 
 It is represented as:
 ```yaml
-terminationGracePeriod: Integer
+terminationGracePeriod: String
 containers: Containers
 toleration: String
 affinity: String
 ```
-- **terminationGracePeriod**: Required integer value. Must be greater than 0. When a game room receives the signal to be deleted, it will take this value (in milliseconds) to be completely deleted;
+- **terminationGracePeriod**: Required string value. Must be greater than 0 and have the unit set, i.e "100s". When a game room receives the signal to be deleted, it will take this value (in seconds) to be completely deleted;
```

Comment:
why do we reference `(in seconds)`? we should allow durations instead

Reply:
Kubernetes only accepts seconds for `terminationGracePeriodSeconds` and we are currently using `google.protobuf.Duration`, which expects only `s` notation: https://protobuf.dev/reference/protobuf/google.protobuf/#duration

<p style="box-sizing: border-box; margin-top: 0px; margin-bottom: 1rem; max-width: 80%; font-weight: 400;">The JSON representation for<span> </span><code style="box-sizing: border-box; font-family: SFMono-Regular, Menlo, Monaco, Consolas, &quot;liberation mono&quot;, &quot;courier new&quot;, monospace; font-size: 13.6px; color: inherit; overflow-wrap: break-word; padding: 0.2em 0.4em; margin: 0px; word-break: normal; background-color: rgba(0, 0, 0, 0.05); border-radius: 0.25rem;">Duration</code><span> </span>is a<span> </span><code style="box-sizing: border-box; font-family: SFMono-Regular, Menlo, Monaco, Consolas, &quot;liberation mono&quot;, &quot;courier new&quot;, monospace; font-size: 13.6px; color: inherit; overflow-wrap: break-word; padding: 0.2em 0.4em; margin: 0px; word-break: normal; background-color: rgba(0, 0, 0, 0.05); border-radius: 0.25rem;">String</code><span> </span>that ends in<span> </span><code style="box-sizing: border-box; font-family: SFMono-Regular, Menlo, Monaco, Consolas, &quot;liberation mono&quot;, &quot;courier new&quot;, monospace; font-size: 13.6px; color: inherit; overflow-wrap: break-word; padding: 0.2em 0.4em; margin: 0px; word-break: normal; background-color: rgba(0, 0, 0, 0.05); border-radius: 0.25rem;">s</code><span> </span>to indicate seconds and is preceded by the number of seconds, with nanoseconds expressed as fractional seconds.</p>